src/engine/cachingreaderworker.cpp

```diff
@@ -143,6 +143,9 @@ void CachingReaderWorker::loadTrack(const TrackPointer& pTrack) {
     // Emit that a new track is loading, stops the current track
     emit trackLoading();
 
+    // Slows down track loading for debug
```

Comment:
Either be more specific "for debugging race conditions while loading multiple tracks in a row" or delete the comment.